project/BuildSettings.scala

```diff
@@ -22,9 +22,9 @@ object BuildSettings {
   // Basic settings for our app
   lazy val basicSettings = Seq[Setting[_]](
     organization  := "com.snowplowanalytics",
-    version       := "0.3.0",
+    version       := "0.3.1",
```

Comment:
1:1:1

Reply:
For non-monorepos we usually bump versions in `Prepared for release` commit